packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -216,15 +216,15 @@ contract GoldToken is
   /**
    * @return The total amount of CELO in existence, including what the burn address holds.
    */
-  function totalSupply() public view returns (uint256) {
+  function totalSupply() external view returns (uint256) {
     return totalSupply_;
   }
 
   /**
    * @return The total amount of CELO in existence, not including what the burn address holds.
    */
   function circulatingSupply() external view returns (uint256) {
-    return totalSupply().sub(getBurnedAmount());
+    return totalSupply_.sub(getBurnedAmount()).sub(balanceOf(address(0)));
```

Comment:
Do we want to do this?

Address zero is not the only address that can potentially cause lost of funds. It kinda makes address zero an unofficial burn address

Reply:
We don't have to but auditors are right that we are treating address(0) differently than any other address - it is not possible to address(0) / cannot increase allowance of address(0) / cannot transfer from address(0). Personally it makes sense to me to include this address but of course we can decide to leave it as it is :-)

Reply:
I think it makes sense to consider address(0) since it is typically considered the default burn address.